my-webxr-app/tests/LoggingUtils.test.tsx

```diff
@@ -0,0 +1,23 @@
+import Rollbar from 'rollbar';
+import { logAppender, LogTableType, rollbarConfig } from '../src/utils/LoggingUtils';
+
+describe('Logger Configuration', () => {
+  const rollbar = new Rollbar(rollbarConfig);
+
+  it('should log message correctly', async () => {
+    // Write log messages to log file
+    rollbar.debug('Test info message from INFO and above log config test');
+    rollbar.info('Test info message from INFO and above log config test');
+    rollbar.warn('Test warn message from INFO and above log config test');
+    rollbar.error('Test error message from INFO and above log config test');
+    rollbar.critical('Test fatal message from INFO and above log config test');
+
+    expect(logAppender.getLogs(LogTableType.info)).toContain('Test info message from INFO and above log config'
+      + ' test');
+    expect(logAppender.getLogs(LogTableType.error)).toContain('Test warn message from INFO and'
+      + ' above'
+      + ' log config test');
+    expect(logAppender.getLogs(LogTableType.test)).toContain('Test error message from INFO and'
+      + ' above log config test');
+  });
```

Comment:
Looks good, testing all levels of logging

Reply:
Sounds good. This was a refactoring of the old logging test, and I see a few things that need to be cleaned up. This also does not work yet since Jest cannot access `import.meta.env` which is how Vite does environment variables